tests/kuttl/tests/backup-complete/00-assert.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu
+  namespace: default
+spec:
+  clusters:
+  - spoke6
+  - spoke2
+  - spoke1
+  - spoke5
+  enable: true
+  backup: true
+  managedPolicies:
+  - policy0-common-config-policy
+  - policy2-common-pao-sub-policy
+  - policy3-common-ptp-sub-policy
+  - policy4-common-sriov-sub-policy
+  remediationStrategy:
+    maxConcurrency: 4
+    timeout: 240
+status:
+  backup:
+    clusters:
```

Comment:
Just realized I missed the backup code in #417. Do you mind updating it along with the test files? Or I can create another PR for the code change only and get it merged first.

Reply:
updated